student.rb

```diff
@@ -10,6 +8,11 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯(ツ)/¯'
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
 end
 puts '########################## Students ruby file #############################'
```

Comment:
- hey, let's kindly remove this console statement as you don't want to leave those logs in your terminal.